src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs

```diff
@@ -114,22 +128,18 @@ private async Task Upload()
             SessionId = sessionId,
             LaunchCode = launchCode,
             BallotCount = ballots.LongLength,
-            BallotImported = 0,
-            BallotSpoiled = 0,
-            BallotDuplicated = 0,
-            BallotRejected = 0,
             SerialNumber = _serialNumber,
-            CreatedBy = UserName
+            CreatedBy = UserName,
         };
 
         long totalCount = 0;
-        long totalInserted = 0;
+        long totalImported = 0;
         long totalDuplicated = 0;
         long totalRejected = 0;
         long totalChallenged = 0;
         long totalSpoiled = 0;
-        ulong startDate = ulong.MaxValue;
-        ulong endDate = ulong.MinValue;
+        var startDate = ulong.MaxValue;
+        var endDate = ulong.MinValue;

```

Comment:
if you are changing this to a var instead of ulong then you also could change the longs above to var by also changing the 0 to a 0L.